pkg/endpointmanager/conntrack.go

```diff
@@ -29,6 +29,16 @@ import (
 const (
 	// MinGcInterval is the minimum garbage collection interval.
 	MinGcInterval int = 5
+
+	// Global string for refering in metrics
+	Global string = "global"
+
+	// Expires string to refering on metrics
+	expires string = "expires"
```

Comment:
Similarly, the comment would be useful if it includes what it actually means (ie it's a metrics value for a scope, and how that scope measures part of the garbage collection).

Reply:
sorry, didn't push without that :-(